code/build.R

```diff
@@ -125,21 +122,20 @@ for(site_name in site_names_to_build) {
 
 # Generate in site metrics  ----------------------------------------------
 
-if(site$is_complete != "no") {
-  
-for(site_name in site_names_to_build) {
-  message("Building for ", site_name)
-  suppressMessages({
-    suppressWarnings({
-      source("code/in-site-metrics.R")
+if (site$is_complete != "no") {
+  for (site_name in site_names_to_build) {
+    message("Building for ", site_name)
+    suppressMessages({
+      suppressWarnings({
+        source("code/in-site-metrics.R")
+      })
     })
-  })
-}
-
+  }
+  
 }
 # Populate site metrics for new site --------------------------------------
 
-if(new_site){
+if (new_site) {
```

Comment:
:+1: in terms of style an improvement.

Reply:
Yeah think this was an accident code format change. I guess if we wanted, we could come up with a script styling guideline.